src/Compilers/CSharp/Test/Emit/Attributes/AttributeTests_Nullable.cs

```diff
@@ -184,9 +185,11 @@ public class Attribute
                 Diagnostic(ErrorCode.WRN_MissingNonNullTypesContextForAnnotation, "?").WithLocation(3, 11),
                 // error CS0518: Predefined type 'System.Byte' is not defined or imported
                 Diagnostic(ErrorCode.ERR_PredefinedTypeNotFound).WithArguments("System.Byte").WithLocation(1, 1),
+                // error CS0518: Predefined type 'System.Byte' is not defined or imported
+                Diagnostic(ErrorCode.ERR_PredefinedTypeNotFound).WithArguments("System.Byte").WithLocation(1, 1),
                 // error CS0518: Predefined type 'System.Int32' is not defined or imported
-                Diagnostic(ErrorCode.ERR_PredefinedTypeNotFound, "").WithArguments("System.Int32").WithLocation(1, 1)
-                );
+                // 
```

Comment:
>//  [](start = 16, length = 3)

Is this empty comment intentional?

Reply:
That was generated by `VerifyDiagnostics`.

---
In reply to: [296930747](https://github.com/dotnet/roslyn/pull/36152#discussion_r296930747) [](ancestors = 296930747)